opensource.md

```diff
@@ -1,3 +1,21 @@
-# Open Source
+# Open Source at MeiliSearch
+Most people will agree that, in its simplest form, Open Source Software (OSS) **is a software that has its source code available**. The Open Source Initiative has a [list of ten criteria](https://opensource.org/osd) defining even more open source, which we agree and follow by using the [MIT license](https://opensource.org/licenses/MIT) for all the source code we release. For us, Open Source is much more than the code.
 
-tbd
\ No newline at end of file
+## Public by default
+We firmly believe that open source isn't just about the code: it's a mindset. It's how we do business and it's how we give back to the tech community.
+
+First, we are including the community in everything we do: **public is the default visibility of our work**. We plan the future of MeiliSearch with the help and feedback of everyone who wants to be involved. It is possible to suggest a [feature idea](https://roadmap.meilisearch.com/tabs/1-under-consideration) to influence the roadmap. We sincerely welcome pull requests, even when it's about a feature that wasn't a priority for us, as a for-profit company. Contributing doesn't only equal code: it's also about making us aware of any problems with our product by creating issues on [GitHub](https://github.com/meilisearch) or being part of our lovely [Slack community](https://slack.meilisearch.com). Note that we will have more documentation on the best ways to contribute to MeiliSearch.
+
+Obviously, we can do even more, so it's why starting from v0.21, we are moving forward with a public [specifications](https://github.com/meilisearch/specifications) process that will close the loop on involving the community and being totally transparent with everything we do product-wise.
```

Comment:
Maybe precise it's the v0.21 of MeiliSearch?

Reply:
You mean for everything? I specified it for the specifications part.

Reply:
I meant this sentence:
> Obviously, we can do even more, so it's why starting from v0.21

I personally get the sentence, I'm not sure an external person understand you meant the v0.21 of MeiliSearch (the core-engine)

Reply:
Oh, ok, make sense, I will change it, thanks